settings.json-example

```diff
@@ -1,4 +1,5 @@
 {
+  "calendar": "primary",
```

Comment:
How about calendarName?  Since the Calendar API also has a concept of an ID.

Reply:
So it's actually a "summary" which is a ridiculous name for this field.  I didn't want to call it an ID and I am reluctant to call it a Name when the API doesn't...